Makefile

```diff
@@ -6,6 +6,9 @@ build:
 	# Use `gradlew.bat build` if on Windows.
 	./gradlew -Penv=production build
 
+test:
+	./gradlew :common-js:test
```

Comment:
If there's only one gradle "project" with tests available then `./gradlew test` should work just as well, plus it'll pick up other tests as they're added to other projects